src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+  const robotKeys = Object.keys(robot);
+  let newRobot = {};
+
+  for (let i = 0; i < robotValues.length; i++) {
+    const type = robotValues[i];
+
+    newRobot[type] = robotKeys[i];
+  }
+
+  for (let i = 0; i < robotValues.length; i++) {
+    for (let n = i + 1; n < robotValues.length; n++) {
+      if (robotValues[i] === robotValues[n]) {
+        newRobot = null;
+      }
+    }
+  }
```

Comment:
you need just one loop, if in your newRobot already has value that you want to add - return null, if not - add it to the new obj